packages/libs/eda/src/lib/map/analysis/hooks/standaloneMapMarkers.tsx

```diff
@@ -316,30 +368,50 @@ export function useStandaloneMapMarkers(
                 },
               ];
 
+        // here's what Bob pointed out
         const count =
           vocabulary != null // if there's an overlay (all expected use cases)
             ? overlayValues.reduce((sum, { count }) => (sum = sum + count), 0)
             : entityCount; // fallback if not
 
+        const bubbleData = [
+          {
+            label: reorderedData[0].label,
+            value: count,
+            color:
+              'color' in reorderedData[0] ? reorderedData[0].color : undefined,
```

Comment:
is this a temp thing until i have the backend put together? not sure i understand..

Reply:
Not sure what in particular you're talking about here, but yeah most likely a temporary thing, lol